### DISCUSSION TITLE
Display glucose values from Datricsae calibration plugin on AW watch?

### DISCUSSION THREAD

**Discussion by Vaughanabe13** [NightscoutFoundation/xDrip#3012] [Ideas]
I have started using the Datricsae plugin under "Advanced Calibration" features. I find it is more accurate for me, and more closely matches the newest Dexcom algorithm that I am running simultaneously on my Tandem X2. The normal algorithm usually differs quite a bit from Dexcom and I find that it is inaccurate compared to the Dexcom. 

Datricsae is not perfect but it seems to be quite a bit better than the standard xdrip+ algorithm or the classic algo. I have both "Plugin plot on graph" and "Use Plugin Glucose" enabled, and therefore I can see the plugin trend and current interpreted value on the main display in the app. I see the "P" indicator on the main graph.

What happens is my watch never receives the plugin glucose values back from the app. I have tried using the phone as the collector, or using the watch as the collector, and they both have the same behavior. I have to physically open the app on my phone, and then it syncs the latest glucose value, but does not resync the previous values. So I get random blips on my watch graph from where it sent the updated plugin value, but the general trend is that of the normal xdrip algo. This is especially frustrating when, for example, the watch trend shows a hypo, but the plugin glucose does not, and in actuality there is no hypo present, confirmed by glucometer.

Basically my request would be to add a way to select which plot gets displayed on the watch so that I can toggle between the original algorithm and the plugin. Or just make it so that when I have a plugin selected and the "Use Plugin Glucose" option is selected, the watch should automatically adjust to use the plugin values.

I'm using the latest nightly build as of the time of this post.

**Comment by revolt5150** [comment]
Did you ever get an answer to this? I'm now in the same situation.

**Comment by Navid200** [comment]
Unfortunately, we don't have a developer who is currently supporting watches.
I'm going to convert this to a discussion.  The conversation can continue there.
